.github/workflows/gh-ci.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
         fail-fast: false
         matrix:
           os: [ubuntu-latest, ]
-          python-version: ["3.10", "3.11", "3.12"]
+          python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
Just testing this out - if it fails we need to exclude full tests on py3.13 deploy

Reply:
It doesn't work due to chemfiles-python (reported upstream).